pkg/asset/internal/templates.go

```diff
@@ -168,13 +161,41 @@ spec:
       - name: ssl-certs-host
         hostPath:
           path: /usr/share/ca-certificates
-      - name: etc-k8s-manifests
-        hostPath:
-          path: /etc/kubernetes/manifests
       - name: secrets
         secret:
           secretName: kube-apiserver
 `)
+	CheckpointerTemplate = []byte(`apiVersion: "extensions/v1beta1"
+kind: DaemonSet
+metadata:
+  name: kube-api-checkpointer
```

Comment:
We may want to name this `checkpoint-installer` to match what it is actually doing. /cc @derekparker